tests/unit/Accordion.js

```diff
@@ -48,9 +48,9 @@ define([
 				(child.headerNode.style.display !== "none" &&
 					((child === target && child.style.display !== "none" &&
 						ac.selectedChildId === target.id && child.open && child.headerNode.open &&
-						$(child).hasClass("d-accordion-open-panel")) ||
+						classList.hasClass(child, "d-accordion-open-panel")) ||
```

Comment:
As I wrote on the delite PR we probably shouldn't be using the deliteful (or delite) methods to assert that deliteful is working, might be better to change to child.classList.contains().   Likewise for all the other tests.